### DISCUSSION TITLE
how to switch frame within one page?

### DISCUSSION THREAD

**Discussion by yomyo** [gdh1995/vimium-c#636] [Q&A] [answered]
some sites have more than one frame,to switch between them have to manually use mouse to click first

for example this site,https://www.glassdoor.com/Job/new-york-jobs-SRCH_IL.0,8_IC1132348.htm
 
how  should i switch between them?

**Comment by gdh1995** [comment, marked as answer]
If you mean `<iframe>`s (https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe), then you may use commands like `nextFrame`, `mainFrame`, and `parentFrame` to switch focus among frames.

You may also use `LinkHints.activate` and select those dark-red hint markers to focus iframes.

But, your example page has no iframes (though I didn't log in), its center&right part is just a scrollable area, so you may just need `LinkHints`.

BTW, maybe https://github.com/gdh1995/vimium-c/discussions/552 helps.
